### ISSUE TITLE
Default parameter setting seems hard to train(offline DV2) 

### ISSUE THREAD

**Issue by wukui-muc** [conglu1997/v-d4rl#6]
Hi, I am currently using the default parameter config setting(dmc_vision,dmc_walker_walk) to train offline DV2 with mixed walker_walk dataset, but the eval return seems to have a big difference compared with the result in the paper. Here are my questions, hope to see your response.
1. In the paper's setting, the world model training epoch is 800 and the agent training epoch is 2400， but in the default configs file,  the world model training epoch is 25001, in code, the world model is trained for 100 steps for each epoch, also, the agent training epoch setting also doesn't match the paper which is far more than paper's setting(2e5 epochs in code, 2400 epochs in paper). Does this need to be modified?
2. I tried to run the different dataset settings, and the eval returns are all far less than the paper's result. For example, in the mixed dataset, the highest eval return only has 240, in the paper, it is nearly 600. Can you share your loss curve or something that can help me to find out the problem?

Here is my  world model training loss curve screenshot(mixed dataset)
![2023-09-11 12-02-20 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/9ccab182-6017-41dd-bb5b-4b6cd939dd02)
![2023-09-11 12-02-28 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/1a0afcb3-118b-47fb-95d7-e0838e3b16f0)
![2023-09-11 12-02-37 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/19de903b-cfeb-408a-be9b-a497bb08963f)

Here is my agent training loss curve (the last checkpoint of the world model) (mixed dataset)
![2023-09-11 12-04-18 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/0f1a2e1e-bedf-49d8-ad65-939ad9b07494)
![2023-09-11 12-04-26 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/9cbc477b-64b3-42b0-a6dc-b8f2bd2b7e39)
![2023-09-11 12-04-35 的屏幕截图](https://github.com/conglu1997/v-d4rl/assets/41557353/3dd2a477-c188-4c20-9889-fc5c4e232dfe)

**Comment by wukui-muc** [comment]
OK, thanks for your clarification, I will try it again

**Comment by ma-rv** [comment]
Hey @conglu1997,

Could you elaborate the calculation of epochs and steps a bit more?

In the code the model is trained for 25k * 100 = 2500k and the agent for 200k * 200 = 40000k steps.

The datasets contain 100k and 200k steps. Lets take 100k for now.
One pass through the whole dataset would require 100k / 64 (batch size) / 50 (traj len) = 31.25 iterations.
If one epoch translates to one full pass of the dataset I have 31.25 * 800 (paper model epochs) = 25000 and 31.25 * 2400 (paper agent epochs) = 75000 steps.

Where is my mistake?

**Comment by conglu1997** [comment]
Your second set of calculations are exactly correct and match what is in the code (e.g. 25k for the model). Why are you multiplying by 100 and 200 in the first set of calculations?

**Comment by ma-rv** [comment]
I wrongly included those extra loops..           
```
for it in range(100):
                model_step.increment()
```

So the model steps are matching. But how do I get from 200k steps (code) to 75k steps (paper) ?

**Comment by ma-rv** [comment]
Perfect! Thanks for your help!
